fpgalib/adc.py

```diff
@@ -38,24 +38,24 @@ class ADC(fpga.FPGA):
     RUN_MODE_DEMOD_AUTO = 4
     RUN_MODE_DEMOD_DAISY = 5
     RUN_MODE_CALIBRATE = 7
-    
+
     @classmethod
     def macFor(cls, boardNumber):
         """Get the MAC address of an ADC board as a string."""
         return cls.MAC_PREFIX + ('0'+hex(int(boardNumber))[2:])[-2:].upper()
-    
+
     @classmethod
     def isMac(mac):
         """Return True if this mac is for an ADC, otherwise False"""
         return mac.startswith(cls.MAC_PREFIX)
-    
+
     # Life cycle methods
-    
+
     @inlineCallbacks
     def connect(self, name, group, de, port, board, build):
         """Establish a connection to the board."""
-        print('connecting to ADC board: %s (build #%d)'\
-            % (self.macFor(board), build))
+        print('connecting to ADC board: {} (build #{})'.format(
+            self.macFor(board), build)
```

Comment:
Missing a closing paren here (that's why the syntax error on line 60).

Personally, I'd suggest not using parens on `print` statements unless you do `from __future__ import print_function` at the top and fix all the `print`s in the module.